src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectViewContainer.tsx

```diff
@@ -147,25 +148,17 @@ const ProjectViewContainer: React.FC<ProjectViewContainerProps> = ({ project, en
   const pageTitle = `${wbsPipe(project.wbsNum)} - ${project.name}`;
 
   return (
-    <>
-      <>
-        <PageBreadcrumbs currentPageTitle={pageTitle} previousPages={[{ name: 'Projects', route: routes.PROJECTS }]} />
-        <Grid container alignItems="center" sx={{ mb: 2 }}>
-          <Grid item>
-            <Typography variant="h4" fontSize={30}>
-              {pageTitle}
-            </Typography>
-          </Grid>
-          <Grid item>
-            <FavoriteProjectButton wbsNum={project.wbsNum} projectIsFavorited={projectIsFavorited} />
-          </Grid>
-          <Grid item sx={{ mx: 0 }} xs>
-            <Grid container direction="row-reverse">
-              <Grid item>{projectActionsDropdown}</Grid>
-            </Grid>
-          </Grid>
-        </Grid>
-      </>
+    <PageLayout title={pageTitle} hidePageTitle>
+      <PageBreadcrumbs currentPageTitle={pageTitle} previousPages={[{ name: 'Projects', route: routes.PROJECTS }]} />
+      <Box sx={{ display: 'flex', flexDirection: ['column', 'row'], justifyContent: 'space-between', gap: 1 }}>
+        <Box sx={{ display: 'flex', flexDirection: 'row', alignItems: 'center' }}>
+          <Typography variant="h4" fontSize={30}>
+            {pageTitle}
+          </Typography>
+          <FavoriteProjectButton wbsNum={project.wbsNum} projectIsFavorited={projectIsFavorited} />
+        </Box>
+        <Grid item>{projectActionsDropdown}</Grid>
+      </Box>
       <ProjectDetailTabs project={project} setTab={setTab} />
       {tab === 0 ? (
```

Comment:
can you make these tabs look like the tabs on the project/CR overview pages, I like this new look with them on the left better